specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -355,7 +454,7 @@
           }
         },
         "operationId": "InvoiceSections_ListByBillingAccountName",
-        "description": "Lists all invoice sections for a user which he has access to.",
+        "description": "Lists all invoice sections for a user which they have access to.",
```

Comment:
Suggestion: "Lists all invoice sections to which a user has access" reads better.

Same comment below. Change "Lists xx for a user which they have access to" to "Lists xx to which a user has access"